module/VuFind/src/VuFind/Db/Entity/AccessToken.php

```diff
@@ -78,15 +79,15 @@ class AccessToken implements AccessTokenEntityInterface
     /**
      * User ID.
      *
-     * @var User
+     * @var int
```

Comment:
Based on the way Doctrine works, this needs to be a User object -- the ORM layer will automatically populate the object based on the relationship annotations below. Code that needs a user ID will have to call $entity->getUser()->getId(). Code that sets the user may need to access to user service if it only has an ID, so it can look up the object that it needs to set.